build.gradle

```diff
@@ -81,6 +81,23 @@ tasks.withType(Test) { task ->
     task.jvmArgs '-Djava.awt.headless=true'  //this prevents awt from displaying a java icon while the tests are running
 }
 
+task testSRA(type: Test) {
+    description = "Run the SRA tests"
+    jvmArgs += '-Dsamjdk.sra_libraries_download=true'
+
+    tags {
+        exclude "slow"
+        exclude "broken"
+    }
+}
+
+
+task FindScalaAndJavaTypes(type: Exec) {
+    description = "Check for Scala outside the scala test dir and java files there."
```

Comment:
"and java files there" -> "and java files in the scala test dir"  (description is a little confusing as-is)